src/invert/laplace/impls/cyclic/cyclic_laplace.hxx

```diff
@@ -44,7 +44,7 @@ class LaplaceCyclic;
  */
 class LaplaceCyclic : public Laplacian {
 public:
-  LaplaceCyclic(Options *opt = nullptr, const CELL_LOC loc = CELL_CENTRE, Mesh *mesh_in = mesh);
+  LaplaceCyclic(Options *opt = nullptr, const CELL_LOC loc = CELL_CENTRE, Mesh *mesh_in = bout::globals::mesh);
```

Comment:
Should this default to `nullptr` for mesh and then get handled in the usual initialiser list way?

Reply:
:+1: I'd left them 'as-is' on the first pass... will update in a new commit.